_data/jobs.yml

```
@@ -1,6 +1,9 @@
 - expires: 2021-09-30
-  location: Rand Corporation, Santa Monica, CA; Pittsburgh, PA; Boston, MA; Washington,
-    D.C.
+  location: Rand Corporation: Santa Monica, CA; Pittsburgh, PA; Boston, MA; Washington, DC
+  name: Research Software Engineer (Level One)
+  url: https://rand.wd5.myworkdayjobs.com/en-US/External_Career_Site/job/Santa-Monica-CA-Greater-Los-Angeles-Area/Research-Software-Engineer--Level-One-_R695
+- expires: 2021-09-30
+  location: Rand Corporation: Santa Monica, CA; Pittsburgh, PA; Boston, MA; Washington, DC
```

...quotes to the rescue!
```suggestion
  location: "Rand Corporation: Santa Monica, CA; Pittsburgh, PA; Boston, MA; Washington, DC"
```